engine/userguide/networking/index.md

```diff
@@ -550,6 +553,34 @@ in default `bridge` network and the
 [linking containers in user-defined networks](work-with-networks.md#linking-containers-in-user-defined-networks)
 for links functionality in user-defined networks.
 
+## Docker and iptables
+
+Linux hosts use a kernel module called `iptables` to manage access to network
+devices, including routing, port forwarding, network address translation (NAT),
+and other concerns. Docker modifies `iptables` rules when you start or stop
+containers which publish ports, when you create or modify networks or attach
+containers to them, or for other network-related operations.
+
+Full discussion of `iptables` is out of scope for this topic. To see which
+`iptables` rules are in effect at any time, you can use `iptables -L`. Multiple
+tables exist, and you can list a specific table, such as `nat`, `prerouting`, or
+`postrouting`, using a command such as `iptables -t nat -L`. For full
+documentation about `iptables`, see
+[netflilter/iptables](https://netfilter.org/documentation/){: target="_blank" class="_" }.
+
+Typically, `iptables` rules are created by an initialization script or a daemon
+process such as `firewalld`. The rules do not persist across a system reboot, so
+the script or utility must run when the system boots, typically at run-level 3
+or directly after the network is initialized. Consult the networking
+documentation for your Linux distribution for suggestions about the appropriate
+way to make `iptables` rules persistent.
+
+Through Docker startup, network creation, container creation, and other actions
+Docker dynamically manages `iptables` rules. In Docker 17.06 and higher, you can
+add rules to a new table called `DOCKER-USER` and Docker will not modify these
```

Comment:
Probably need a slight rewording; docker already wouldn't touch your rules, however, the problem was that it was not possible to reliably put your own rules _before_ Docker's rules. The new feature makes it so that Docker puts it's rules _after_ the rules defined in `DOCKER-USER`.

I'm bad at iptables, so may be good to have someone from networking confirm this.

If we don't have this written down yet (as a follow up), we may want to describe;

- `DOCKER` chain
- `DOCKER-INGRESS` chain
- `DOCKER-ISOLATION` chain
- that `--iptables=false` can disable iptables management (but IIRC, cannot be combined with swarm mode, which always requires some iptables rules to be managed)